internal/pkg/cli/app_init.go

```diff
@@ -315,6 +315,8 @@ An application is a collection of containerized services that operate together.`
   /code $ copilot app init test
   Create a new application with an existing domain name in Amazon Route53.
   /code $ copilot app init --domain example.com
+  Create a new application with an existing IAM policy for roles with a permissions boundary.
```

Comment:
I probably don't know what I'm talking about but what do you think of this alternative
```suggestion
  Create a new application with an existing IAM policy as the permissions boundary for roles.
```
My only concern is that people parsing it as "...with an existing IAM policy for roles" i.e. the roles' permissions are granted by the IAM policy, instead of "limited/bound".

Reply:
Your version is much better! Thanks!